src/python/grapl-http-service/integration_tests/BUILD

```diff
@@ -0,0 +1,4 @@
+python_tests(
+    name="integration",
```

Comment:
Strictly speaking, this name isn't required here. It will default to the name of the directory.